pandas/plotting/_core.py

```diff
@@ -3537,7 +3658,8 @@ def scatter(self, x, y, s=None, c=None, **kwds):
             ...                       c='species',
             ...                       colormap='viridis')
         """
```

Comment:
Could you add an example here? I'd say just repeat the previous one with `s='species'`, even if it doesn't make a lot of sense (will need to convert species to an ordered categorical).